src/gwt/src/org/rstudio/core/client/virtualscroller/virtualscroller.js

```diff
@@ -192,8 +193,14 @@ var VirtualScroller;
       return this.scrollerEle.scrollTop < 4;
     },
 
-    _scrolledToBottom: function() {
-      return Math.abs(this.scrollerEle.scrollHeight - this.scrollerEle.offsetHeight - this.scrollerEle.scrollTop) < 4;
+    scrolledToBottom: function() {
+      return Math.abs(this.scrollerEle.scrollHeight - this.scrollerEle.offsetHeight - this.scrollerEle.scrollTop) < 50;
```

Comment:
Could `this.scrollerEle` ever be `null` here? (Mainly asking because `scrollToBottom` has a check for this)

Reply:
possible! added a null check, thanks!